core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -165,6 +165,85 @@ describe('Admin backend api service', () => {
     flushMicrotasks();
   }));
 
+  it('should send SVGs to the backend.', fakeAsync(() => {
+    var successResponse = {
+      result: 'successfully updated'
+    };
+    var latexToSvgMapping = {
+      exp_id1: {
+        latex_string1: {
+          file: new Blob(),
+          dimensions: {
+            encoded_height_string: '4d456',
+            encoded_width_string: '3d467',
+            encoded_vertical_padding_string: '0d234'
+          },
+          latexId: '3rmYki9MyZ'
+        }
+      },
+      exp_id2: {
+        latex_string2: {
+          file: new Blob(),
+          dimensions: {
+            encoded_height_string: '3d456',
+            encoded_width_string: '5d467',
+            encoded_vertical_padding_string: '0d234'
+          },
+          latexId: '4rm6ki9MsZ'
+        }
+      }
+    };
+    var expectedPayload = {
+      latexMapping: {
+        exp_id1: {
+          latex_string1: {
+            dimensions: {
+              encoded_height_string: '4d456',
+              encoded_width_string: '3d467',
+              encoded_vertical_padding_string: '0d234'
+            },
+            latexId: '3rmYki9MyZ'
+          }
+        },
+        exp_id2: {
+          latex_string2: {
+            dimensions: {
+              encoded_height_string: '3d456',
+              encoded_width_string: '5d467',
+              encoded_vertical_padding_string: '0d234'
+            },
+            latexId: '4rm6ki9MsZ'
+          }
+        }
+      }
+    };
+
+    abas.sendMathSvgsToBackend(latexToSvgMapping);
+    let req = httpTestingController.expectOne(
+      '/adminmathsvghandler');
+    var requestBody = req.request.body;
+    expect(requestBody instanceof FormData).toBeTruthy();
+    var rawImageSentToBackend = null;
+    var image1 = null;
+    var image2 = null;
+    var payLoadSentoBackend = null;
+    requestBody.forEach((value, key) => {
+      if (key === '3rmYki9MyZ') {
```

Comment:
Prefer not having if else in test blocks. Can you repeat the code instead?

Reply:
Yeah I'll make a follow-up PR to adress your and @DubeySandeep comment. Thanks

Reply:
Please file an issue for any deferred changes, and link to it from here. Do this as a matter-of-course each time.

Once you have filed the issue and added it here, I will merge this PR.

Reply:
@seanlip, I've created an issue. #10128